diff --git a/colors/phcs.vim b/colors/phcs.vim
index d02a727..7ba7d00 100644
--- a/colors/phcs.vim
+++ b/colors/phcs.vim
@@ -14,7 +14,6 @@ endif

 set t_Co=256
 let g:colors_name = "phcs"
-
 hi Normal                                              guifg=#e8e6e3 guibg=#101410 guisp=#101410 gui=NONE ctermfg=254 ctermbg=233 cterm=NONE
 hi Comment                                             guifg=#2ea303 guibg=NONE guisp=NONE gui=NONE ctermfg=34 ctermbg=NONE cterm=NONE
 hi Todo                                                        guifg=#f5faf6 guibg=#1f6603 guisp=#1f6603 gui=NONE ctermfg=194 ctermbg=22 cterm=NONE
@@ -29,7 +28,7 @@ hi Special                                            guifg=#c75b5b guibg=#000000 guisp=#000000 gui=NONE ctermfg=167 c
 hi SpecialChar                                 guifg=#c75b5b guibg=#000000 guisp=#000000 gui=NONE ctermfg=167 ctermbg=NONE cterm=NONE
 hi cspecialcharacter                   guifg=#c75b5b guibg=#000000 guisp=#000000 gui=NONE ctermfg=167 ctermbg=NONE cterm=NONE
 hi cformat                                             guifg=#c75b5b guibg=#000000 guisp=#000000 gui=NONE ctermfg=167 ctermbg=NONE cterm=NONE
-hi Identifier                                  guifg=#808bed guibg=NONE guisp=NONE gui=NONE ctermfg=105 ctermbg=NONE cterm=NONE
+hi Identifier                                  guifg=#2ea303 guibg=NONE guisp=NONE gui=NONE ctermfg=119 ctermbg=NONE cterm=NONE
 hi Directory                                   guifg=#808bed guibg=NONE guisp=NONE gui=NONE ctermfg=105 ctermbg=NONE cterm=NONE
 hi Type                                                        guifg=#808bed guibg=NONE guisp=NONE gui=NONE ctermfg=105 ctermbg=NONE cterm=NONE
 hi Typedef                                             guifg=#ff8bff guibg=NONE guisp=NONE gui=NONE ctermfg=213 ctermbg=NONE cterm=NONE
@@ -55,27 +54,27 @@ hi PMenuThumb                                       guifg=#0a0a0a guibg=#a4a6a8 guisp=#a4a6a8 gui=NONE ctermfg=232
 hi TabLine                                             guifg=#000000 guibg=#305a8a guisp=#305a8a gui=NONE ctermfg=NONE ctermbg=24 cterm=NONE
 hi TabLineSel                                  guifg=#ffffff guibg=#7097e0 guisp=#7097e0 gui=NONE ctermfg=15 ctermbg=68 cterm=NONE
 hi TabLineFill                                 guifg=#ffffff guibg=#383838 guisp=#383838 gui=NONE ctermfg=15 ctermbg=237 cterm=NONE
-hi StatusLine                                  guifg=#0a0a0a guibg=#4f6d8f guisp=#4f6d8f gui=bold ctermfg=232 ctermbg=60 cterm=bold
+hi StatusLine                                  guifg=#e8e6e3 guibg=#4f6d8f guisp=#4f6d8f gui=NONE ctermfg=254 ctermbg=60 cterm=NONE
 hi StatusLineNC                                        guifg=#0a0a0a guibg=#4f6d8f guisp=#4f6d8f gui=NONE ctermfg=232 ctermbg=60 cterm=NONE
-hi StatusLineTerm                              guifg=#0a0a0a guibg=#58a878 guisp=#44855e gui=bold ctermfg=232 ctermbg=72 cterm=bold
-hi StatusLineTermNC                            guifg=#0a0a0a guibg=#58a878 guisp=#44855e gui=NONE ctermfg=232 ctermbg=72 cterm=None
+hi StatusLineTerm                              guifg=#e8e6e3 guibg=#58a878 guisp=#44855e gui=NONE ctermfg=254 ctermbg=72 cterm=NONE
+hi StatusLineTermNC                            guifg=#0a0a0a guibg=#58a878 guisp=#44855e gui=NONE ctermfg=232 ctermbg=72 cterm=NONE
 hi LineNr                                              guifg=#8b8bcd guibg=#2e2e2e guisp=#2e2e2e gui=NONE ctermfg=104 ctermbg=236 cterm=NONE
 hi CursorLineNr                                        guifg=#b3ddf2 guibg=#2e2e2e guisp=#2e2e2e gui=NONE ctermfg=153 ctermbg=236 cterm=NONE
 hi NonText                                             guifg=#8b8bcd guibg=NONE guisp=NONE gui=NONE ctermfg=104 ctermbg=NONE cterm=NONE
 hi VertSplit                                   guifg=#b9b9b9 guibg=#3e3e5e guisp=#3e3e5e gui=NONE ctermfg=250 ctermbg=60 cterm=NONE
 hi DiffDelete                                  guifg=#ffffcd guibg=#290606 guisp=#290606 gui=NONE ctermfg=230 ctermbg=52 cterm=NONE
-hi DiffAdd                                             guifg=NONE guibg=#183608 guisp=#183608 gui=NONE ctermfg=NONE ctermbg=22 cterm=bold
-hi DiffText                                            guifg=NONE guibg=#383807 guisp=#383807 gui=NONE ctermfg=NONE ctermbg=58 cterm=bold
+hi DiffAdd                                             guifg=NONE guibg=#183608 guisp=#183608 gui=NONE ctermfg=NONE ctermbg=22 cterm=NONE
+hi DiffText                                            guifg=NONE guibg=#383807 guisp=#383807 gui=NONE ctermfg=NONE ctermbg=58 cterm=NONE
 hi DiffChange                                  guifg=NONE guibg=#101410 guisp=#101410 gui=NONE ctermfg=NONE ctermbg=233 cterm=NONE
 hi Folded                                              guifg=#cfcfcd guibg=#4b208f guisp=#4b208f gui=NONE ctermfg=252 ctermbg=54 cterm=NONE
 hi FoldColumn                                  guifg=#8b8bcd guibg=#2e2e2e guisp=#2e2e2e gui=NONE ctermfg=104 ctermbg=236 cterm=NONE
 hi Error                                               guifg=NONE guibg=#ce4e4e guisp=#ce4e4e gui=NONE ctermfg=NONE ctermbg=167 cterm=NONE
-hi ErrorMsg                                            guifg=#0f0f0f guibg=#a80303 guisp=#a80303 gui=bold ctermfg=233 ctermbg=124 cterm=bold
-hi WarningMsg                                  guifg=#090a0a guibg=#c9a204 guisp=#c9a204 gui=bold ctermfg=232 ctermbg=178 cterm=bold
+hi ErrorMsg                                            guifg=#ffffff guibg=#a80303 guisp=#a80303 gui=NONE ctermfg=15 ctermbg=124 cterm=NONE
+hi WarningMsg                   guifg=#090a0a guibg=#c9a204 guisp=#c9a204 gui=NONE ctermfg=232 ctermbg=178 cterm=NONE
 hi Cursor                                              guifg=#131414 guibg=#f5fafa guisp=#f5fafa gui=NONE ctermfg=233 ctermbg=195 cterm=NONE
 hi lCursor                                             guifg=#131414 guibg=#f5fafa guisp=#f5fafa gui=NONE ctermfg=233 ctermbg=195 cterm=NONE
 hi CursorIM                                            guifg=#131414 guibg=#f5fafa guisp=#f5fafa gui=NONE ctermfg=233 ctermbg=195 cterm=NONE
-hi CursorLine                                  guifg=NONE guibg=#1d1d1f guisp=#1d1d1f gui=NONE ctermfg=NONE ctermbg=234 cterm=NONE
+"hi CursorLine                                 guifg=NONE guibg=#1d1d1f guisp=#1d1d1f gui=NONE ctermfg=NONE ctermbg=234 cterm=NONE
 hi SpellBad                                            guifg=NONE guibg=#ce4e4e guisp=#ce4e4e gui=NONE ctermfg=NONE ctermbg=167 cterm=NONE
 hi WhitespaceEOL                               guifg=#ffffff guibg=#ce4e4e guisp=#ce4e4e gui=NONE ctermfg=15 ctermbg=167 cterm=NONE
 hi SpecialKey                                  guifg=#ab60ed guibg=NONE guisp=NONE gui=NONE ctermfg=135 ctermbg=NONE cterm=NONE
@@ -83,9 +82,9 @@ hi Character                                  guifg=#f0ac6d guibg=#101410 guisp=#101410 gui=NONE ctermfg=215
 hi Question                                            guifg=#ffcd00 guibg=NONE guisp=NONE gui=NONE ctermfg=220 ctermbg=NONE cterm=NONE
 hi Title                                               guifg=#e5f06e guibg=NONE guisp=NONE gui=NONE ctermfg=228 ctermbg=NONE cterm=NONE
 hi mbenormal                                   guifg=#ffffff guibg=NONE guisp=NONE gui=NONE ctermfg=15 ctermbg=NONE cterm=NONE
-hi mbechanged                                  guifg=#e5f06e guibg=NONE guisp=NONE gui=NONE ctermfg=228 ctermbg=NONE cterm=NONE
+hi mbechanged                                  guifg=#ffcd00 guibg=NONE guisp=NONE gui=NONE ctermfg=220 ctermbg=NONE cterm=NONE
 hi mbevisiblenormal                            guifg=#00ff8b guibg=NONE guisp=NONE gui=NONE ctermfg=48 ctermbg=NONE cterm=NONE
-hi mbevisiblechanged                   guifg=#e5f06e guibg=NONE guisp=NONE gui=NONE ctermfg=228 ctermbg=NONE cterm=NONE
+hi mbevisiblechanged                   guifg=#ffcd00 guibg=NONE guisp=NONE gui=NONE ctermfg=220 ctermbg=NONE cterm=NONE
 hi taglisttagname                              guifg=#808bed guibg=NONE guisp=NONE gui=NONE ctermfg=105 ctermbg=NONE cterm=NONE
 hi SpecialComment                              guifg=#f5faf6 guibg=#1f6603 guisp=#1f6603 gui=NONE ctermfg=194 ctermbg=22 cterm=NONE
 hi Debug                                               guifg=#ffcd00 guibg=NONE guisp=NONE gui=NONE ctermfg=220 ctermbg=NONE cterm=NONE

